server/routes/checkAnswers.js

```diff
@@ -21,7 +22,109 @@ module.exports = function Index({ formService, authenticationMiddleware }) {
     '/:bookingId',
     asyncMiddleware(async (req, res) => {
       const errors = req.flash('errors')
-      res.render('pages/check-answers', { data: res.locals.formObject, bookingId: req.params.bookingId, errors })
+      const { bookingId } = req.params
+
+      // offender details
+      const offenderDetail = await offenderService.getOffenderDetails(res.locals.user.token, bookingId)
+
+      // retrieve form inputs
+      // eslint-disable-next-line
+      const loadForm = async req => {
+        const { form_response: formObject = {}, id: formId } = await formService.getFormResponse(
+          req.user.username,
+          bookingId
+        )
+        return { formId, formObject }
+      }
+      const { formObject } = await loadForm(req, res)
+
+      // parent obj to send to view
+      const viewObject = formObject.incident || {}
+      viewObject.newIncident = viewObject.newIncident || {}
+
+      const offenderLocation = await offenderService.getLocation(
+        res.locals.user.token,
+        viewObject.newIncident.locationId
+      )
+
+      // new incident object
+      const newIncident = {
+        'Offender name': offenderDetail.displayName,
+        'Offender number': offenderDetail.offenderNo,
+        Location: offenderLocation.description,
+        'Use of force planned': viewObject.newIncident.forceType,
+        'Staff involved': toTitleCase(convertArrayOfObjectsToString(viewObject.newIncident.involved)),
+        Witnesses: toTitleCase(convertArrayOfObjectsToString(viewObject.newIncident.witnesses)),
+      }
+      viewObject.newIncident = newIncident
+      viewObject.offenderDetail = offenderDetail
+
+      // details object. Initilise to previous values or empty object to prevent 'undefined' errors
```

Comment:
As discussed it would be worth breaking up this large method and extracting each section into separate simple functions, so you end up with no mutation, maybe something like this: 
```
const viewModel = {
   details: buildDetails(),
  relocationAndInjuries: buildRelocationAndInjuries(),
  evidence: buildEvidence(),
  // ... etc..
}
```

Reply:
will do this as a separate ticket together with renaming Yes to yes etc

Reply:
I've refactored the function into smaller units that each build up the newIncident, details, relocation and evidence objects. 
You may be wondering why I've needed lines 31 to 34 of checkAnswers.js but I could only get the functionality to work by including that code.